Sources/ArcGISToolkit/FloorFilter/FloorFilterViewController.swift

```diff
@@ -0,0 +1,511 @@
+        guard let floorFilterStackView = floorFilterStackView else { return }
+        addCornerRadiusBasedOnPlacement()
+        
+        // First remove the Close Button and Site Button from the stack and then based on placement add the Buttons.
+        floorFilterStackView.removeArrangedSubview(closeBtn)
+        floorFilterStackView.removeArrangedSubview(siteBtn)
+        floorFilterStackView.setNeedsLayout()
+        
+        if (expansionDirection == .down) {
+            // Place the Site Button at the top of the Floor Filter View.
+            // The Levels List will be unchanged and remain as the second element on the Floor Filter view.
+            // Place the Close Button at the bottom of the list.
+            floorFilterStackView.insertArrangedSubview(siteBtn, at: 0)
+            floorFilterStackView.insertArrangedSubview(closeBtn, at: 2)
+            floorFilterStackView.setNeedsLayout()
+        } else {
+            // Place the Close Button at the top of the Floor Filter View.
+            // The Levels List will be unchanged and remain as the second element on the Floor Filter view.
+            // Place the Site Button at the bottom of the list.
+            floorFilterStackView.insertArrangedSubview(closeBtn, at: 0)
+            floorFilterStackView.insertArrangedSubview(siteBtn, at: 2)
+            floorFilterStackView.setNeedsLayout()
+        }
+    }
+}
+
+extension FloorFilterViewController: UITableViewDataSource, UITableViewDelegate {
+    
+    
+    public func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return state == .partiallyExpanded ? 1 : viewModel.visibleLevelsInExpandedList.count
+    }
+    
+    public func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "levelCell", for: indexPath)
+        let levels = viewModel.visibleLevelsInExpandedList
+        
+        cell.textLabel?.font = levelFont
+        cell.textLabel?.adjustsFontSizeToFitWidth = true
+        cell.textLabel?.textAlignment = .center
+        setTableViewCellCornerRadius(cell: cell)
+        
+        // If the Floor Filter state is fully expanded then set the title of the cell based on the levels list.
+        // Otherwise set the title of cell to the selected level short name.
+        if (state == .fullyExpanded) {
+            cell.textLabel?.text = levels[indexPath.row].shortName
+        } else {
+            cell.textLabel?.text = viewModel.selectedLevel?.shortName
+                ?? viewModel.defaultLevel(for: viewModel.selectedFacility)?.shortName
+        }
+        
+        let visibleLevelVerticalOrder = levels.first(where: \.isVisible)?.verticalOrder
+        if let text = cell.textLabel?.text,
+           levels.contains(where: { level in
+            level.verticalOrder == visibleLevelVerticalOrder && level.shortName == text
```

Comment:
The indentation is off here. One very helpful feature of Xcode is the ability to automatically re-indent the selected lines (Editor > Structure > Re-Indent ⌃I). 
```suggestion
               level.verticalOrder == visibleLevelVerticalOrder && level.shortName == text
```